Dockerfile.bm-inventory-disconnected

```
@@ -0,0 +1,27 @@
+# [TODO] - remove this line, once we are ready to use openshift-installer from the release
+FROM quay.io/yshnaidm/openshift-installer AS openshift-installer
+FROM quay.io/ocpmetal/ignition-manifests-and-kubeconfig-generate:stable as config-gen
+
+FROM fedora:31
+
+ARG GIT_REVISION
+ARG WORK_DIR=/data
+
+LABEL "git_revision"=${GIT_REVISION}
+
+RUN dnf install -y python pip libvirt-devel
```

Try to unite dockerfile steps as much as possible to reduce layers and the image size
e.g. 
```
RUN dnf install -y python pip libvirt-devel && pip install boto3 botocore
```

---

Cleaning up after dnf will help
```suggestion
RUN dnf install -y python pip libvirt-devel && dnf clean all
```
Don't think installing pip in the same layer will reduce image size

---

@eranco74 Each layer counts - doesn't matter its content.

---

@YuviGold that is just wrong.
```
$ cat dockerfile.1l
FROM fedora:31
RUN dnf install -y python pip libvirt-devel && pip install boto3
$ cat dockerfile.2l
FROM fedora:31
RUN dnf install -y python pip libvirt-devel
RUN pip install boto3
$ docker images | grep layer
2layer                                                        latest                                     eed2166e776e        8 minutes ago       554MB
1layer                                                        latest                                     7a902c214e64        10 minutes ago      554MB

```